include/command_macros_base.h

```diff
@@ -17,9 +17,9 @@
 #define CMD_W(a) (a)
 
 #ifdef __GNUC__
-#define CMD_F(a) {.f = (a)}
+#define CMD_F(a) .f = (a)
```

Comment:
I added the curly braces on the commands which use this macro instead of here. see [include/z64cutscene_commands.h](https://github.com/zeldaret/mm/pull/514/files#diff-4c6608c322a04d0fd0e43abd946330aee847d32cb6fddef7f4220b9fc71f7f2a)